examples/compound/types.go

```diff
@@ -0,0 +1,15 @@
+package compound
+
+import (
+	"github.com/nakji-network/connector/examples/compound/ctoken"
+
+	"google.golang.org/protobuf/proto"
+)
+
+var TopicTypes = map[string]proto.Message{
```

Comment:
can generate these using `GenerateTopicFromProto`, shouldn't need to define these for every connector